Cards/Cards.css

```diff
@@ -0,0 +1,212 @@
+body {
+    font-family: sans-serif;
+  }
+  @import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+  
+  :root {
+    --card-BG: rgb(255, 255, 255);
+    --primary-color: rgb(238, 27, 36);
+    --secondary-color: rgb(117, 117, 117);
+    --tertiary-color: rgb(117, 117, 117, 0.8);
+    --hover-color: rgba(238, 27, 36, 0.2);
+  }
+  
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  
+  .wrapper {
+    height: 100vh;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+    flex-wrap: wrap;
+    margin-top: 1rem;
+  }
+  .card-container {
+    width: 40rem;
+    display: flex;
+    flex-direction: column;
+    padding-bottom: 0.5rem;
+    gap: 0.4rem;
+    border-radius: 0.4rem;
+    background-color: var(--card-BG);
+    box-shadow: 4px 4px 15px rgba(0, 0, 0, 0.15);
+  }
+  .card-top-section {
+    display: flex;
+    gap: 1rem;
+    align-items: center;
+    position: relative;
+  }
+  
+  .img-tint {
+    width: 40rem;
+    height: 12.5rem;
+    position: absolute;
+    left: 0;
+    top: 0;
+    background-color: rgba(0, 0, 0, 0);
+  }
+  .img-tint:hover {
+    background-color: rgba(68, 67, 67, 0.08);
+  }
+  
+  .image-box {
+    height: 12.5rem;
+    width: 10rem;
+  }
+  
+  .image-box img {
+    height: 100%;
+    width: 100%;
+    object-fit: fill;
+    border-top-left-radius: 0.4rem;
+    border-top-right-radius: 0.4rem;
+  }
+  
+  .card-details {
+    font-size: 0.8rem;
+  }
+  
+  .card-details h2 {
+    margin-top: 0.4rem;
+    color: var(--secondary-color);
+  }
+  
+  .card-details h4 {
+    margin-top: 0.4rem;
+    font-size: 0.6rem;
+    color: var(--tertiary-color);
+  }
+  .card-bottom-section {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    margin-left: auto;
+    padding-right: 0.5rem;
+  }
+  .primary-solid {
+    background-color: black;
+    border: 1.5px solid black;
+    border-radius: 0.3rem;
+    color: white;
+    padding: 0.8rem 1.5rem;
+  }
+  
+  .primary-outline {
+    background-color: white;
+    border: 1.5px solid black;
+    border-radius: 0.3rem;
```

Comment:
For border radius try using px bcoz generally you don't want to increase border radius the screen size increase

Reply:
Yeah.I agree. Made the changes as suggested. Thanks